website/docs/docs/user-interface/dashboard.md

```diff
@@ -96,3 +100,19 @@ Clicking on a tag pill for a provider version tag will make the following menu a
 
 - Copy tag URL - selecting this menu item will copy the URL of the application version tag into the clipboard.
 - Copy tag name - selecting this menu item will copy the name of the tag into the clipboard.
+
+## Matrix 
+
+"Matrix" displays pacts and verifications of the selected integration.
+
+&nbsp;
+
+![Matrix Screen](/ui/matrix.png)
+
+&nbsp;
+
+### Matrix Filters
+Filter your pacts and verifications of selected integration with consumer and provider's version, branch, environment and tag.
```

Comment:
"Filter the consumer versions and provider versions of the selected integration by version number, branch, environment or tag".